corehq/toggles.py

```diff
@@ -1852,3 +1852,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_CUSTOM,
     [NAMESPACE_DOMAIN]
 )
+
+
+ENABLE_ICDS_DASHBOARD_MANUAL_UPDATE = StaticToggle(
+    'enable_icds_dashboard_manual_update',
+    'Enable updating ICDS dashboard manual for specific users',
```

Comment:
At the moment it looks like this feature flag is gating both viewing the new manual and updating the dasbhoard manual. Would it be possible to avoid adding this feature flag and restricting the upload view to super users?